implementations/rust/ockam/ockam_node/src/context.rs

```diff
@@ -136,13 +153,24 @@ impl Context {
 
     async fn new_context_impl(&self, addr: Address) -> Result<Context> {
```

Comment:
Hmm, yea. I agree that this is confusing. Although having had a look at the way we construct `Context` I didn't see a super obvious alternative.

We could have `Context::new` and `Context::new_bare` but the functions would be almost identical (except for the async_drop field). Maybe it would be enough to pass in a value for `async_drop_sender` to `Context::new` so that it becomes more obvious how the two types of Context are different?